intelligence/ts/create/index.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+import { init } from '@flwr/create-flwr/utils/createProject.js';
+import inquirer from 'inquirer';
+import chalk from 'chalk';
+
+const starterKits = ['node-ts', 'node-js', 'web-ts'];
+
+(() => {
+  console.log(chalk.yellow(`Welcome to Flower Intelligence!`));
+  inquirer
+    .prompt([
+      {
+        type: 'input',
+        name: 'projectName',
+        message: 'Enter your project name',
+        default: 'my-fi-project',
```

Comment:
```suggestion
        default: 'awesome-flower-intelligence-app',
```

I would suggest du use some default name which contains "Flower Intelligence"

Reply:
Yes!